cli/commands/benchmark-web-vitals.mjs

```diff
@@ -151,6 +160,7 @@ function getParamsFromOptions( opt ) {
 		showVariance: Boolean( opt.showVariance ),
 		cpuThrottleFactor: null,
 		networkConditions: null,
+		windowViewport: { width: 960, height: 700 }, // Viewport similar to @wordpress/e2e-test-utils 'large' configuration.
```

Comment:
It makes sense to have this parameter, but I wonder whether we should additionally have more than just this one default, to simplify usage.

I'm thinking about our typical `mobile` vs `desktop` differentiation. Maybe we could allow passing special string values `mobile` and `desktop` for the `--window-viewport` argument, and in that case rely on a specific default for that viewport type that we set?

I wonder what PageSpeed Insights uses, maybe we could align with that?

Reply:
Good idea. That did cross my mind, but I didn't implement it. I picked the `360x800` and `1920x1080` since they are the most common mobile and desktop dimensions currently. In 1f3b1c556043283edb6825baae46e485e938bb0e I've implemented the `mobile` and `desktop` aliases, and I've set their default values to correspond to what Lighthouse uses.